commands/bootstrap/inputs.go

```diff
@@ -596,7 +613,47 @@ func initializeInputs(inputFilePath, resourcesPath string, configuration config.
 		}
 	}
 
+	if inputsPath == "" {
+
+		bSlice, err := yaml.Marshal(inputValues)
+
+		inputsPathOut := deploymentName + ".yaml"
```

Comment:
If the user calls `./yorc bootstrap` without the `-n` option, then deploymentName is empty here
(it will be set to a timestamped value in deployTopology() that is called later).

So a file inputsaves/.yaml will be created.

To avoid this, We may move the code setting deploymentName to a timestamped value from deployTopology() to initializeInputs() so that it is not empty when we reach the line here.